modules/debezium_database_user/postgres_user.tf

```diff
@@ -0,0 +1,45 @@
+locals {
+  grant_command = <<EOF
+docker run -e PGPASSWORD="${var.granter_password}" --rm --entrypoint="" postgres:${var.postgres_version}-alpine \
+  psql \
+  --host ${var.database_address} \
+  --port ${var.database_port} \
+  --username ${var.granter_username} \
+  --dbname "${var.database_name}" \
+  --command "CREATE ROLE ${var.debezium_username} with LOGIN PASSWORD
+    '${var.debezium_password}' VALID UNTIL 'infinity' IN ROLE rds_replication;
+    GRANT USAGE ON SCHEMA public TO debezium;
+    GRANT SELECT, INSERT, DELETE ON TABLE ${var.debezium_events_table} TO ${var.debezium_username};
+    CREATE PUBLICATION debezium_public_events FOR TABLE ${var.debezium_events_table} WITH (publish = 'insert'); "
```

Comment:
This was put in as a bit of a hack.  We should probably move this line to a different file.

Reply:
What do you have in mind? Have it's own resource?